README.md

```diff
@@ -14,7 +14,9 @@ bundler install
 or use:
 
 ```shell
-bundle install --path vendor/bundle
+
+bundle config set path 'vendor/bundle'
```

Comment:
```suggestion
gem install -i vendor/bundle bundler
bundle config --local path 'vendor/bundle'
bundle install
```
issues here:
1. my `gem install` would also try to install into a system Ruby dir, so needs a user path as well.
2. `bundle config` directly takes the variable name, no `set`. To set this specifically for this repo only, passing `--local` seems sensible.